x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -151,6 +155,18 @@ export function createBulkExecutionEnqueuerFunction({
       );
     }
 
+    if (
+      await hasReachedTheQueuedActionsLimit(
```

Comment:
should we try to queue as many as we can and return a list of IDs that couldn't be enqueued?

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/164632/commits/539da5a45f47c0c95a3cbfdd7fdbcbcd70ad2330